depends/packages/boost.mk

```diff
@@ -1,6 +1,6 @@
 package=boost
 $(package)_version=1_71_0
-$(package)_download_path=https://dl.bintray.com/boostorg/release/$(subst _,.,$($(package)_version))/source/
+$(package)_download_path=https://boostorg.jfrog.io/artifactory/main/release/$(subst _,.,$($(package)_version))/source/
```

Comment:
Why not https://github.com/boostorg/boost/archive/refs/tags/boost-1.71.0.tar.gz ?

Reply:
Wrong sha256 and it is named boost-boost, and so is the folder.

They say this is going to be the new stable home, ref:
https://github.com/boostorg/boost/issues/502

Sorry for not including this in the first comment.

Reply:
> Wrong sha256 

I'd say "different", not "wrong" :)

> They say this is going to be the new stable home, ref:
> [boostorg/boost#502](https://github.com/boostorg/boost/issues/502)

To be [precise](https://github.com/boostorg/boost/issues/502#issuecomment-817954776):
> That should be the stable home; _but we're still working on it_.

Another input to think about it: https://www.qt.io/press/the-qt-company-strengthens-its-market-leader-position-with-acquisition-of-froglogic-gmbh?hsLang=en&hsCtaTracking=ec89c874-5e14-41a0-b58d-77d585bd9e02%7C2406baa6-6d9e-480e-a0ad-e58bab8676e3

... and Qt has controversial relations with the open-source community.

Reply:
True, different sha256. I did not want to suggest a link to a file with a different sha256 for security and trust reasons.

Yes, they are working on it. Bintray is closing, and have scheduled down-time. Next time the 26th. https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

Getting it from github will require a new sha256 and additional changes to boost.mk.
The file you get from the github download link is  `boost-boost-1.71.0.tar.gz`

If you feel github is the more appropriate way to get it, I have no problem with that.

At the time I made this PR, building did not work. (bintray 403.)
According to the blog-post, it will happen again the 26th.